src/mmrelay/k8s_utils.py

```
@@ -0,0 +1,769 @@
+_MISSING_CONFIG_KEYS_MSG = "Missing required config keys: {keys}"
+
+logger = get_logger(__name__)
+
+_SERIAL_CONTAINER_DEVICE_PATH = "/dev/ttyUSB0"
+
+
+def _get_storage_classes_from_kubectl() -> list[tuple[str, bool]] | None:
+    """Return storage class names and default flags using kubectl, if available."""
+    try:
+        result = subprocess.run(
+            ["kubectl", "get", "storageclass", "-o", "json"],
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)